content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -218,6 +224,85 @@ Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number
 dogstatsd_so_rcvbuf: 26214400
 ```
 
+### Over UDS (Unix Domain Socket)
+
+#### Linux
+
+For UDS sockets, Linux is internally buffering datagrams in a queue if the
+reader is slower than the writer. The size of this queue represents the maximum number
+of datagrams that Linux will buffer per socket. This value can be queried with the following command:
+
+```bash
+sysctl net.unix.max_dgram_qlen
+```
+
+If the value is < 512, you can increase it to 512 or more using this command:
+
+```bash
+sysctl -w net.unix.max_dgram_qlen=512
+```
+
+Add the following configuration to `/etc/sysctl.conf` to make this change permanent:
+
+```conf
+net.unix.max_dgram_qlen = 512
+```
+
+In the same manner, the `net.core.wmem_max` could be incremented to 4MiB to improve
+client writing performances:
+
+```conf
+net.core.wmem_max = 4194304
+```
+
+Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number in `datadog.yaml`:
+
+```yaml
+dogstatsd_so_rcvbuf: 4194304
+```
+
+Please note that setting these values will need a reboot of the host.
```

Comment:
is this true for all the options mentioned here? 🤔

Pretty sure at least `net.core.wmem_max` is taken into account at runtime without a reboot?

Can be addressed in a follow-up PR in any case

Reply:
No you're right, I've first added this here to be cautious but I read yesterday in kubernertes doc that net sysctls doesn't need a reboot! I'll remove that.